doc/api/cluster.md

```diff
@@ -626,6 +626,8 @@ values are `"rr"` and `"none"`.
     (Default=`process.argv.slice(2)`)
   * `silent` {Boolean} whether or not to send output to parent's stdio.
     (Default=`false`)
+  * `stdio` {Array} Configures the stdio of forked processes. When this option
+    is provided, it overrides `silent`.
```

Comment:
It will throw in `child_process.fork()` if there is no IPC. I'll update these docs accordingly.